src/widgets/WebexMeeting/WebexMeeting.jsx

```diff
@@ -25,27 +25,38 @@ export default class WebexMeetingWidget extends Component {
     });
     this.state = {
       adapterConnected: false,
+      meeting: null,
     };
     this.adapter = new WebexSDKAdapter(webex);
+    this.meetingSubscription = null;
   }
 
   async componentDidMount() {
     await this.adapter.connect();
     // Once adapter connects, set our app state to ready.
     this.setState({adapterConnected: true});
+
+    this.meetingSubscription = this.adapter.meetingsAdapter
+      .createMeeting(this.props.meetingDestination)
+      .subscribe((meeting) => this.setState({meeting}));
   }
 
   async componentWillUnmount() {
+    if (this.meetingSubscription) {
+      this.meetingSubscription.unsubscribe();
```

Comment:
It won't harm to do this, but `createMeeting` emits once and completes so there is no need to unsubscribe

Reply:
@lalli-flores This is due to my confusion between `createMeeting` and `getMeeting`.

You're correct, `createMeeting` emits a single message, while `getMeeting` emits a new meeting on each meeting event, keeping the subscriber up to date with the meeting object. The widget will need this updating to respond to status changes (eg: when the user leaves the meeting, the widget should hide the roster).

Can we return from `createMeeting` the same thing that `getMeeting` does? It seems that whenever a meeting is created, `getMeeting` is called immediately anyway.

If you prefer to keep `createMeeting` just for meeting creation (and it makes sense to have it so), then I will kindly ask @maloun96 to re-add the `.pipe(getMeeting)` line I asked him to remove yesterday:)

Reply:
I can see why it can be confusing, however, I would like to keep `createMeeting` strictly about meeting creation. I know there are use cases out there where you may create a meeting, but not join it right away.

So yes, you would need to re-add .pipe(getMeeting) 😜